### ISSUE TITLE
[Question] Chitu v6 board and Chitu Wifi module 

### ISSUE THREAD

**Issue by KingBain** [luc-github/ESP8266#702]
On the stock chitu v6 board(common board in TronXY X5sa pro printers) the pinout is this:

![image](https://user-images.githubusercontent.com/367922/147604083-3ae31150-5148-4936-9611-68fd7aaec2b9.png)

 The pinout 
 
![image](https://user-images.githubusercontent.com/367922/147604042-b512e3b4-237c-4c17-a93a-43bb0ae9fe2a.png)

From factory, Pin 5 is pulled down to ground. According to the documentation here, 

Pin 5 is GPIO0 on the Tronxy Wifi modules, which is basically an ESP01 with 1MB of from.

so if Pin 5 in pulled down to ground, this means the board will never boot up because ESP8266 is being put into upload mode. 

![image](https://user-images.githubusercontent.com/367922/147604251-8979274f-6d78-451d-ae7f-31f48b5cd7a0.png)

Am I understanding that correctly ?

If I soldered a 10K resistor between pin 5 and pin 8, would that pull pin 5 up ?

Also from factory Pin 4 (CH_PD) is getting 5v .... how bad is that for the ship ?

**Comment by luc-github** [comment]
are you sure they use a ESP8266-01 connector ? put GPIO to GND is not normal at start
also I see it have 5V output which is not supported by any ESP chips, ESP8266 have a tolerance but it is not recommended at all

it looks more a custom connector - for custom esp8266-01 board

**Comment by Thomaskru** [comment]
Hi!
I have also this board (tronxy x5sa-2e). ESP8266-01 is not 1to1 compatible with this pinout.

You can use male/female Dupont wires to lift the esp of this socket. Connect pins
(Chitu -> ESP8266-01)
1 GND -> 1 GND
2 WIFI_TX -> 2 TX
7 WIFI_RX -> 7 RX
8 3V3 -> 8 3V3
 via wires.

Additionaly you must shorten 3V3 & EN (Pin 8 & 4) on the esp8266-01 to force boot. 

You can then use pins gpio0 & gpio2 from the Chitu board for other things like  bltouch or something else. (Marlin only).

ESP32 is also possible with these RX/TX Pins and when it has support for 5V you can use pin 4 from chitu board to power ESP32.

**Comment by KingBain** [comment]
I ended up just bending the GPIO 0 pin out of the way and inserting the wifi module into the socket on to the board. 

The ESP3d then successfully booted.

I also found this on a Marlin forum talking about the v6 pinouts. I had to translate it from Chinese to english 
![image](https://user-images.githubusercontent.com/367922/147685995-3cdb224c-1e12-4291-ad6e-ec7ff4c4dc73.png)


it looks like they're aware of the v5 issue... I guess they just run the chip hot :) 

I read somewhere that the ESP8266 is 5v tolerant ... we'll see :D

{edit}

here is the wifi module that I ordered https://www.aliexpress.com/item/4000862952158.html?spm=a2g0o.9042311.0.0.1dcc4c4d77gCOx
